### ISSUE TITLE
display events which lasts more than one day on all-day slot

### ISSUE THREAD

**Issue by AinaRav** [fullcalendar/fullcalendar#7258]
### Checklist

- [X] I've already searched through [existing tickets](https://github.com/fullcalendar/fullcalendar/issues)
- [X] Other people will find this feature useful

### Is this feature for a specific connector (React/Angular/etc)?

No connector in particular

### Feature Description

I would like to display events which lasts more than one day on all-day slot.
As of now the event is displayed like this:

![image](https://user-images.githubusercontent.com/38721068/227509857-c6421ae2-7dbd-448c-b94b-60e22d1da09e.png)

I would like it to be displayed like this:

![image](https://user-images.githubusercontent.com/38721068/227510202-9d707a07-64d6-45c5-96b8-922ca63016c0.png)

**Comment by acerix** [comment]
Do you mean having it show as both a timed event and an all-day event?

You could set `allDay: true` to change it to an all-day event:

https://codepen.io/acerix/pen/wvYwpyj?editors=001

**Comment by AinaRav** [comment]
> Do you mean having it show as both a timed event and an all-day event?
> 
Yes. Since the event lasts several days, I think it would be consistent to have it shown also on the all-day slot.
